web/src/services/api-register.ts

```diff
@@ -64,6 +64,7 @@ export default {
   get_product_version_endpt: (version :string) => `/api/${version}/product_version`,
   get_health_components_endpt: (version :string) => `/api/${version}/system/health/components`,
   get_health_resources_endpt: (version :string) => `/api/${version}/system/health/resources`,
+  get_health_cluster_endpt: (version :string) => `/api/${version}/system/health/node`,
```

Comment:
No need to hard code node here. You can use another parameter for resource.
Ex: get_health_cluster_endpt: (version :string, resource: string) => `/api/${version}/system/health/${resource}`

Reply:
Updated endpoint implementation